Project.toml

```diff
@@ -20,16 +20,16 @@ Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 TimerOutputs = "a759f4b9-e2f1-59dc-863e-4aeb61b1ea8f"
 
 [compat]
-BlackBoxOptim = "^0.5, 0.6"
-Distances = "^0.8, ^0.9, ^0.10"
-LatinHypercubeSampling = "^1.2"
-NearestNeighbors = "^0.4"
-Parameters = "^0.12"
-ScatteredInterpolation = "^0.3"
-StaticArrays = "^0.12, 1.0"
-StatsBase = "^0.32, 0.33"
-TimerOutputs = "^0.5"
-julia = "^1.3"
+BlackBoxOptim = "0.5, 0.6"
+Distances = "0.8, 0.9, 0.10"
+LatinHypercubeSampling = "1"
+NearestNeighbors = "0.4"
+Parameters = "0.12"
+ScatteredInterpolation = "0.3"
+StaticArrays = "0.12, 1"
+StatsBase = "0.32, 0.33, 0.34"
```

Comment:
This is at least the major change.

Reply:
Confirmed it works with StatsBase v0.34, so take your pick on how you want this represented, but at least the `^` are unncessary because it default carrots.